src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -93,8 +94,8 @@ public sealed class Driver
         /// <summary>
         /// The job name option.
         /// </summary>
-        private static readonly OptionInfo<string?> JobNameOption = new OptionInfo<string?>(
-            ImmutableList.Create("--job-name"), default, "The name of the submitted job.");
+        private static readonly OptionInfo<string> JobNameOption = new OptionInfo<string>(
+            ImmutableList.Create("--job-name"), "", "The name of the submitted job.");
```

Comment:
Doesn't matter much, but String.Empty is considered cleaner.

Reply:
Changed to `string.Empty`.